elasticsearch/files/sysconfig

```diff
@@ -1,6 +1,3 @@
-{%- if 'JAVA_HOME' not in sysconfig.keys() %}
-JAVA_HOME=/usr/lib/java
```

Comment:
Why default to /usr/lib/java?

Reply:
This was the old default I guess. This merge request is removing that. 

Thanks for the review by the way !